### PR TITLE
soften links

### PR DESCRIPTION
**Pull request by russelltsherman**

### CODE REVIEW

**Comment by russelltsherman** [comment]
now with less ENVVAR

File: class.ThemePage.php [hunk 1]
```diff
@@ -46,7 +37,7 @@ function add_links()
                     $this->addLink('About', $this->secureUrl);
                 }
             }
-            $this->addLink('Logout', 'http://profiles.burningflipside.com/logout.php');
+            $this->addLink('Logout', $this->profilesUrl.'/logout.php');
```
Comment (pboyd04) [inline class.ThemePage.php]: $this->logoutUrl

File: class.ThemePage.php [hunk 2]
```diff
@@ -10,15 +10,6 @@ function __construct($title)
         $root = $_SERVER['DOCUMENT_ROOT'];
         $script_dir = dirname(__FILE__);
         $this->theme_root = substr($script_dir, strlen($root));
-        $this->secureUrl = $this->settings->getGlobalSetting('secure_url', 'https://secure.burningflipside.com');
-        $this->secureMenu = $this->settings->getGlobalSetting('secure_menu', array(
```
Comment (pboyd04) [inline class.ThemePage.php]: Where does this get calculated for this page?

Comment (russelltsherman) [reply class.ThemePage.php]: this appears to be redundant.. 
isn't it rendering the same as is generated by the plugin code in the secure page class?

Comment (pboyd04) [reply class.ThemePage.php]: That code saves to a local variable. Checkout my changes in #4. That should fix most of this...

**Review by pboyd04 — APPROVED** [review]

**Pull request merged**